IFC4x3/Properties/i/IsLiftingBracket_0Py8wuJJrAwPXDvufAW9rN/Documentation.md

```diff
@@ -0,0 +1 @@
+Is it a connection between two different heights? Yes/no
```

Comment:
I am no expert, but can the definition be improved?

Reply:
I have improved it! adb5fdd751ea322bf0649365b080a81b2a458a62